packages/common/util.ts

```diff
@@ -3,7 +3,7 @@ export interface _Id {
 }
 
 export type WithoutId<T extends {}> = Omit<T, 'id'>
-export type WithId<T extends {}> = T & {id: string}
+export type WithId<T extends {}> = T & { id: string, [key: string]: any }
```

Comment:
What should this be? I do this so I can access WithId fields by string value.

Reply:
I don't think you need to change the type here.  This will allow anything to be a key-value, which isn't right.  Using `any` is just circumventing the type system which we've very carefully built.  We know what the fields are.  You should not be doing this.